### ISSUE TITLE
[BUG]: beginner pack gave wrong items

### ISSUE THREAD

**Issue by vincywo** [typical-developers/oaklands-feedback#1141]
### Roblox Username

vincy_io

### What version did this issue occur in?

1.69.2

### Describe the issue in detail.

i purchased the beginner pack thinking i would get the refined axe and pickaxe but i only got the iron ones. 

### Provide some images / videos of the issue in action.

i dont even know how to put this into a picture 💀

### Before Submitting

- [X] I have confirmed that nobody else reported this issue.
- [X] The details of this report are legitimate.
- [X] I understand that I cannot be reimbursed here for in-game content I have lost.

**Comment by 1Peekaboo1** [comment]
???
This is not a bug, you were clearly shown in the graphics what you are getting.

**Comment by vincywo** [comment]
yeah i was. it said i was getting the refined items not the iron ones

**Comment by 1Peekaboo1** [comment]
![изображение](https://github.com/user-attachments/assets/be20bffe-17c6-4600-bfd9-76f746502993)
...?
